src/components/more-from-blogs/more-from-blogs.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../card";
+
+import * as styles from './more-from-blogs.module.scss';
+
+const MoreBlogsSection = ({ data, refMoreFrom, threeBlogsSectionTitle }) => {
```

Comment:
please rename the file o rename the component, but both should have the same name